docs/Makefile

```diff
@@ -3,7 +3,9 @@ DIAGRAMS := \
 	attestation/amd/sev/certchain.dot.png \
 	attestation/amd/sev/process.msc.png \
 	attestation/ibm/pef/certchain.dot.png \
-	attestation/ibm/pef/process.msc.png
+	attestation/ibm/pef/process.msc.png \
+	attestation/intel/sgx/sgx_certchain.dot.png \
+	attestation/intel/sgx/sgx_process.msc.png
```

Comment:
This Makefile was replaced by a cargo-make Makefile.

These entries now belong in `docs/Makefile.toml` and the notable difference here is that now the final rendered image format extension is omitted from the initial list.